capture/project_library/project.py

```diff
@@ -101,16 +102,23 @@ def get_waves(self, start: Optional[int] = None, end: Optional[int] = None):
         """ Get waves from project.
         """
         if self.project_cfg.type == "cw":
-            return self.project.waves[start:end]
+            if (start is not None) and (end is not None):
+                return self.project.waves[start:end]
+            elif start is not None:
+                return self.project.waves[start]
+            else:
+                return self.project.waves
```

Comment:
Question: why don't you need to handle the case where `end` is provided but not start? You now seem to then always provide the full data set.

Reply:
If somebody just calls `get_waves(0)` (i.e. give me the wave at DB entry 0) then start is set to 0 as it is the first parameter. This is equivalent to `get_waves(start=0)`. End is in this case None.
I don't think that anybody will try `get_waves(end=0)`.

If this sounds like a non-optimal decision from my side, I could throw an error if somebody tries to do `get_waves(end=0)`

Reply:
Ah, now I understand. This sounds good to me. Thanks for explaining!